includes/Modules/Analytics.php

```diff
@@ -1385,8 +1420,12 @@ protected function create_analytics_site_data_request( array $args = array() ) {
 			$dimension_filter->setOperator( 'EXACT' );
 			$args['page'] = str_replace( trim( $this->context->get_reference_site_url(), '/' ), '', esc_url_raw( $args['page'] ) );
 			$dimension_filter->setExpressions( array( rawurldecode( $args['page'] ) ) );
+			$dimension_filters_clauses[] = $dimension_filter;
```

Comment:
This is incorrect – only the `Google_Service_AnalyticsReporting_DimensionFilterClause` should be added to this list, not the `Google_Service_AnalyticsReporting_DimensionFilter` instance (as it is in the current version of the file on the base branch).

Reply:
Yes, you're correct.